TeachingRecordSystem/src/TeachingRecordSystem.Core/Events/UserActivationEventChanges.cs

```diff
@@ -0,0 +1,7 @@
+namespace TeachingRecordSystem.Core.Events;
+
+[Flags]
+public enum UserActivationEventChanges
```

Comment:
Don't think we need this - there will only ever be one option.

Reply:
removed